background/update.js

```diff
@@ -15,11 +16,21 @@ var updater = {
   MAYBE_EDITED: 'may be locally edited',
   SAME_MD5: 'up-to-date: MD5 is unchanged',
   SAME_CODE: 'up-to-date: code sections are unchanged',
+  SAME_VERSION: 'up-to-date: version is unchanged',
   ERROR_MD5: 'error: MD5 is invalid',
   ERROR_JSON: 'error: JSON is invalid',
+  ERROR_VERSION: 'error: version is older than installed style',
 
```

Comment:
@tophf Unrelated to this PR, but it appears that the majority of the history entries come directly from this reference. Should they be localized?

Reply:
Uhm yes, we'll expose all of them later.